app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -31,7 +31,6 @@ def plan(capsule_content, options = {})
               end
               plan_action(ConfigureCapsule, capsule_content, environment, content_view, repository)
               sync_repos_to_capsule(capsule_content, repository_ids)
-              plan_action(RemoveOrphans, :capsule_id => capsule_content.capsule.id)
```

Comment:
They were complaining, but in that was when the orphan cleanup wasn't being performed at all on the fpc servers :)